### PR TITLE
test pr3

### PR DESCRIPTION
**Pull request by shandr**

### CODE REVIEW

**Comment by shandr** [comment]
> you should add spaces

ok, I'll add them

File: myfile3.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python3
+
+a = [1,2,3]
+  for l in a:
+print(l)
```
Comment (ozakhariash-ledger) [inline myfile3.py:5]: still not correct

Comment (shandr) [reply myfile3.py:5]: What should be changed?

Comment (shandr) [reply myfile3.py:5]: test 3

**Pull request merged**